packages/@aws-cdk/aws-secretsmanager/README.md

```diff
@@ -15,35 +14,15 @@ import * as secretsmanager from '@aws-cdk/aws-secretsmanager';
 ```
 
 ### Create a new Secret in a Stack
-
 In order to have SecretsManager generate a new secret value automatically,
 you can get started with the following:
 
-```ts
-// Default secret
-const secret = new secretsmanager.Secret(this, 'Secret');
-
-// Using the default secret
-new iam.User(this, 'User', {
-  password: secret.secretValue,
-});
-
-// Templated secret
-const templatedSecret = new secretsmanager.Secret(this, 'TemplatedSecret', {
-  generateSecretString: {
-    secretStringTemplate: JSON.stringify({ username: 'user' }),
-    generateStringKey: 'password',
-  },
-});
-
-// Using the templated secret
-new iam.User(this, 'OtherUser', {
-  userName: templatedSecret.secretValueFromJson('username').toString(),
-  password: templatedSecret.secretValueFromJson('password'),
-});
-```
+[example of creating a secret](test/integ.secret.lit.ts)
 
-[see also this example of creating a secret](test/integ.secret.lit.ts)
+The `Secret` construct does not allow specifying the `SecretString` property
+of the `AWS::SecretsManager::Secret` resource (as this will almost always
```

Comment:
Can we add some pointers/example on how to create a secretsmanager secret and reference it in a CDK app?

(Subsequent commit)